reference/configuration/framework.rst

```diff
@@ -266,7 +266,7 @@ reverse proxy. See :doc:`/deployment/proxies`.
 ide
 ~~~
 
-**type**: ``string`` **default**: ``null``
+**type**: ``string`` **default**: ``%env(default::SYMFONY_IDE)%``
```

Comment:
I think we should add a `.. versionadded:: 6.1` directive before or after note ?

Reply:
It is already on line 323 😉